docs/pages/enroll-resources/application-access/cloud-apis/aws-console.mdx

```diff
@@ -369,6 +369,12 @@ In this step, you will define a Teleport role that confers access to the
    Management Console or make requests to AWS APIs with the `ExampleReadOnlyAccess`
    role.
 
+   <Admonition type="important" title="AWS roles and App labels">
+   Role needs permissions for `aws_role_arns` and `app_labels` for AWS access to work.
+
+   If a role has `aws_role_arns` defined but not `app_labels`, attempting to access the AWS Console will result in a `Not Found` error.
```

Comment:
i would love to learn about the problem you run into. if there is no `app_labels`, the user should not even see the aws console app in WebUI? how did you end up seeing "Not Found"? is it a product bug?

the wording here feels more like a troubleshooting scenario to me. should it be moved there? and we could add more comments in the yaml definition above to explain what `app_labels` and `aws_role_arns` and that they need to appear together